### ISSUE TITLE
ModuleNotFoundError: No module named 'pyimod02_importers'

### ISSUE THREAD

**Issue by ewertonhm** [DataDog/datadog-sync-cli#173]
**Overview of issue:**
Using the windows release version 0.5.0, we get the following error while trying to run:
``` powershell
PS C:\Users\Ewerton\Downloads> .\datadog-sync-cli-windows-amd64.exe
[20960] Module object for pyimod02_importers is NULL!
Traceback (most recent call last):
  File "PyInstaller\loader\pyimod02_importers.py", line 22, in <module>
  File "pathlib.py", line 14, in <module>
  File "urllib\parse.py", line 40, in <module>
ModuleNotFoundError: No module named 'ipaddress'
Traceback (most recent call last):
  File "PyInstaller\loader\pyiboot01_bootstrap.py", line 17, in <module>
ModuleNotFoundError: No module named 'pyimod02_importers'
[20960] Failed to execute script 'pyiboot01_bootstrap' due to unhandled exception!
```
**Steps to reproduce the issue:**
1. Download the datadog-sync-cli-windows-amd64.exe from releases
2. Run it with powershell 

**Describe the results you received:**
Get the following error:
``` powershell
[29556] Module object for pyimod02_importers is NULL!
Traceback (most recent call last):
  File "PyInstaller\loader\pyimod02_importers.py", line 22, in <module>
  File "pathlib.py", line 14, in <module>
  File "urllib\parse.py", line 40, in <module>
ModuleNotFoundError: No module named 'ipaddress'
Traceback (most recent call last):
  File "PyInstaller\loader\pyiboot01_bootstrap.py", line 17, in <module>
ModuleNotFoundError: No module named 'pyimod02_importers'
[29556] Failed to execute script 'pyiboot01_bootstrap' due to unhandled exception!
```

**Describe the results you expected:**
Should see the help message

**Additional information you deem important (e.g. issue happens only occasionally):**
Tried on two diferente computers, same issue,
tried versions 0.5.0 and 0.4.2

 Running from source with python worked fine.

**Comment by ewertonhm** [comment]
Looks live a pyinstaller problem, based on the following issue: 
#https://github.com/pyinstaller/pyinstaller/issues/7692

using pyinstaller>=5.12 should fix it

**Comment by skarimo** [comment]
Thanks for the report and debugging the issue! Just released `0.5.1` with the fix.
